apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css

```diff
@@ -120,12 +120,71 @@
 	border-radius: 50%;
 }
 
+.circle-gradient4 {
+	position: absolute;
+	display: none;
+	background: linear-gradient(to right, #0085ff 0%, #00f0ff 100%);
```

Comment:
Do we have these colors in variables.css?

Reply:
We weren't at the time when I written this, will use css variables now